### DISCUSSION TITLE
output enriched terms

### DISCUSSION THREAD

**Discussion by safrikDut** [egeulgen/pathfindR#74] [General]
Hi! Firstly, I would like to express my gratitude for pathfindR which has been very useful for my PhD project. I have a question regarding the output from term enrichment table . There is a column called "non_signif_Snw_Genes" which are visually marked as blue. What are these genes? I have looked at the help of PathfindR and sadly I could not find info about it. 
I also have a question regarding term_visualization. How can I change colors and visualization settings for the output graph? Because of the style of other figures I need to change it to a bar graph. Could you please help me with this?

**Comment by egeulgen** [comment]
hey @safrikDut,
Thank you for your kind words. To answer your questions:
1-  `non_signif_Snw_Genes` are genes in the active subnetwork (that led to the enrichment) that are not input genes but genes linked to the input genes in the protein-protein interaction network
2- Unfortunately, you cannot change the output of term_visualization. You would have to write a separate function to achieve the task
